src/app/_components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Typography, Button } from "@mui/material";
+import React from "react";
+
+const ForgotPassword = (): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="column">
+      <Typography mb={1} variant="h5" sx={{ fontWeight: "bold" }}>
+        Password reset
+      </Typography>
+      <Typography mb={5} sx={{ color: "text.secondary" }}>
+        Your password has been successfully reset. click confirm to set a new
+        password
+      </Typography>
+
+      <Button
+        variant="contained"
+        style={{ backgroundColor: "#648DDB" }}
+        id="confirm-reset-password"
+      >
```

Comment:
rebase with `dev` to get the theme changes so that here you can just use the default mui button without additional style